src/Admin/AbstractAdmin.php

```diff
@@ -3208,6 +3210,14 @@ public function canAccessObject($action, $object)
         return $this->hasAccess($action, $object);
     }
 
+    /**
+     * @return string[]
+     */
+    protected function configureExportFields(): array
```

Comment:
This method and signature does not follow the same idea as the other configure: parameter with the initial config and empty body

Reply:
This is not the rule for array https://github.com/sonata-project/SonataAdminBundle/blob/3.x/src/Admin/AbstractAdminExtension.php#L92